libdeno/binding.cc

```diff
@@ -409,7 +412,15 @@ v8::MaybeLocal<v8::Module> ResolveCallback(v8::Local<v8::Context> context,
   v8::Context::Scope context_scope(context);
 
   int ref_id = referrer->GetIdentityHash();
-  std::string referrer_filename = d->module_filename_map_[ref_id];
+  auto range = d->module_info_map_.equal_range(ref_id);
+  std::string referrer_filename = "";
+  for (auto it = range.first; it != range.second; ++it) {
+    if (it->second.second == referrer) {
```

Comment:
Here the `==` is comparing v8 persistent objects? Please add a comment to that effect.

Reply:
It is overloaded to compare the value inside the handle.
https://denolib.github.io/v8-docs/include_2v8_8h_source.html#l00487

I'll add a comment on this.

Reply:
Nice docs